### ISSUE TITLE
Does OpenNMT-tf support extra word features input?

### ISSUE THREAD

**Issue by howardyclo** [OpenNMT/OpenNMT-tf#89]
OpenNMT Lua version supports extra word features concatenated with token with "｜" delimiter and the decoder can also make predictions of word features.
I'm wondering if Tensorflow version has this functionality?

**Comment by howardyclo** [comment]
Ok, I found that we can incorporate word features with [parallel inputs](http://opennmt.net/OpenNMT-tf/data.html#parallel-inputs). But it seems like we cannot make the decoder predict those features like Lua version? (a.k.a multi-task learning)

**Comment by guillaumekln** [comment]
You are correct, additional word features are only supported on the source side using parallel inputs.
